pkg/constants/constants.go

```diff
@@ -39,4 +39,10 @@ const (
 	DefaultGatewayListenerProtocol v1alpha2.ProtocolType = "HTTP"
 	DefaultFunctionServicePort     v1alpha2.PortNumber   = 80
 	DefaultFuncPort                v1alpha2.PortNumber   = 8080
+
+	WasmEdgeWorkloadRuntimeName = "wasmedge"
+	WasmEdgeRuntimeClassName    = "openfunction-crun"
+	WasmEdgeBuildStrategyName   = "wasmedge"
```

Comment:
No need to add `wasmedge` buildStrategy?

Reply:
It can also be processed by passing parameters to `buildah` strategy. but our `fn.Spec.Build.Params` donot support array values. need another pr.
```
  status:
    completionTime: "2023-02-15T10:14:59Z"
    conditions:
    - lastTransitionTime: "2023-02-15T10:14:59Z"
      message: 'the Build is not registered correctly, build: builder-j7x66-build-5p5f8,
        registered status: False, reason: WrongParameterValueType'
      reason: BuildRegistrationFailed
      status: "False"
      type: Succeeded
```

Reply:
> It can also be processed by passing parameters to `buildah` strategy. but our `fn.Spec.Build.Params` donot support array values. need another pr.

Another PR to do what?
Can we simply change the wasmEdgeBuildStrategy to `buildah`?

Reply:
build wasmedge program need this parameter:
```
parameters:
  - defaults:
    - annotation=module.wasm.image/variant=compat-smart
    description: The values for the args in the Dockerfile. Values must be in the
      format KEY=VALUE.
    name: build-args
    type: array
```